aiosmtpd/tests/test_starttls.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import unittest
+import pkg_resources
+
+from aiosmtpd.smtp import SMTP as SMTPProtocol
+from email.mime.text import MIMEText
+from smtplib import SMTP
+from .test_smtp import Controller as OldController, Sink
```

Comment:
Generally I prefer absolute imports, but I think I'll just fix this one up after landing the branch.